intelmq/lib/message.py

```diff
@@ -1,66 +1,101 @@
-import json 
+# -*- coding: utf-8 -*-
+"""
+Messages are the information packages in pipelines.
+
+Use MessageFactory to get a Message object (types Report and Event).
+"""
+import json
 import hashlib
-import base64
 from intelmq import HARMONIZATION_CONF_FILE
 from intelmq.lib import utils
 import intelmq.lib.harmonization
 import intelmq.lib.exceptions as exceptions
 
 
+harm_config = utils.load_configuration(HARMONIZATION_CONF_FILE)
```

Comment:
Why did you put this line here? We evaluate this line when we wrote it and we found that line there will be load multiple times...less perfomance...   dont remember well but I think that was one of the reasons